js/sidemenu.js

```diff
@@ -0,0 +1,12 @@
+let openSideMenu = document.querySelector('.fa-bars');
+openSideMenu.addEventListener('click', function() {
+    document.getElementById('side-menu').style.width = '100%';
+    // When clicking the hamburger icon for the first time, the side menu appears abruptly. From the second time onwards it shows up from the side. Is there a way to make the menu show up from the side from the first click?
```

Comment:
There is more than likely a way to make it work that way! It's hard for me to visualize what it's doing until I fire it up in my local.  Want to video chat sometime this week and we can look at it together?